backend/services/implementations/testService.ts

```diff
@@ -0,0 +1,32 @@
+import MgTest, { Test } from "../../models/test.model";
+import { CreateTestRequestDTO, SchoolTestDTO } from "../../types";
+import { getErrorMessage } from "../../utilities/errorUtils";
+import logger from "../../utilities/logger";
+import { ITestService } from "../interfaces/testService";
+
+const Logger = logger(__filename);
+
+class TestService implements ITestService {
+  /* eslint-disable class-methods-use-this */
+  async createTest(test: CreateTestRequestDTO): Promise<SchoolTestDTO> {
```

Comment:
nit: maybe rename `SchoolTestDTO` to `TestResponseDTO` for consistency with other services / more clarity

Reply:
+1